tools/http-fault-injector/sample-servers/net/http-server/Program.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Buffers;
+using System.Net;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+
+namespace Azure.Sdk.Tools.HttpFaultInjector.HttpServerSample
+{
+    public class Program
+    {
+        public static void Main(string[] args)
+        {
+            new WebHostBuilder()
+                .UseKestrel(options =>
+                {
+                    options.Listen(IPAddress.Any, 5000);
+                    options.Listen(IPAddress.Any, 5001, listenOptions =>
+                    {
+                        listenOptions.UseHttps("testCert.pfx", "testPassword");
+                    });
+                })
+                .Configure(app => app.Run(async context =>
+                {
+                    Console.WriteLine($"Request: {context.Request.Path}");
```

Comment:
nit: you can still do [`app.UseHttpLogging()`](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/http-logging/?view=aspnetcore-8.0) to have logging configurable, controllable and exportable